masterlist.yaml

```
@@ -438,6 +444,8 @@ plugins:
         name: 'Trackers Alliance: The Vulture'
     group: *bgsCreationsGroup
 
+
+
 ###### Verified Creations ######
```

There are some extra newlines here:

```suggestion

###### Verified Creations ######
```